### ISSUE TITLE
Crew refuses to dive the submarine after using Emergency Blow.

### ISSUE THREAD

**Issue by GuyIncognito0** [DotModGroup/ColdWaters-DotMod#65]
I used the Emergency Blow to evade some torpedoes and the helmsman refused to follow commands to keep the submarine from breaching the surface and to make depth 500' or even dive at all. The submarine does not require compressed air to take on ballast. It is a system for emergencies and requiring it to refill during combat while under attack is something the player should be able to override.

**Comment by PolarisESP** [comment]
Yes, i in the reality the submarines can dive when they use the emergency blow.

**Comment by MateusJK15** [comment]
Submarines use the snorkel mast to pump air to the ballast tanks in order to surface, the compressed air banks are only for emergencies.
